frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/backgroundables/MergeCurrentBranchFastForwardOnlyBackgroundable.java

```diff
@@ -0,0 +1,36 @@
+package com.virtuslab.gitmachete.frontend.actions.backgroundables;
+
+import com.intellij.openapi.project.Project;
+import git4idea.commands.GitCommand;
+import git4idea.commands.GitLineHandler;
+import git4idea.repo.GitRepository;
+import lombok.Getter;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class MergeCurrentBranchFastForwardOnlyBackgroundable extends GitCommandUpdatingCurrentBranchBackgroundable {
+
+  @Getter
+  private final String targetBranchName;
+
+  public MergeCurrentBranchFastForwardOnlyBackgroundable(
+      Project project,
+      GitRepository gitRepository,
+      String taskTitle,
+      String targetBranchName) {
+    super(project, gitRepository, taskTitle);
+    this.targetBranchName = targetBranchName;
+  }
+
+  @Override
+  protected String getOperationName() {
+    return "Fast-forward merge";
```

Comment:
I18n => move to bundle (here and in other places)?

Reply:
Done both here and in `PullCurrentBranchFastForwardOnlyBackgroundable`